packages/hash/frontend/src/theme/components/navigation/MuiMenuItemThemeOptions.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  Components,
+  Theme,
+  menuItemClasses,
+  dividerClasses,
+  listItemTextClasses,
+  listItemIconClasses,
+  listItemAvatarClasses,
+} from "@mui/material";
+
+export const MuiMenuItemThemeOptions: Components<Theme>["MuiMenuItem"] = {
+  defaultProps: {
+    disableRipple: true,
+    disableTouchRipple: true,
+  },
+  styleOverrides: {
+    root: ({ ownerState, theme }) => ({
+      borderRadius: "4px",
+      padding: theme.spacing(1, 1.5),
+      ...theme.typography.smallTextLabels,
+
+      "&:hover": {
+        backgroundColor: theme.palette.gray[20],
+        color: theme.palette.gray[80],
+      },
+
+      [`& .${listItemTextClasses.primary}`]: {
+        ...theme.typography.smallTextLabels,
+        fontWeight: 500,
+        color: theme.palette.gray[80],
+      },
+
+      [`& .${listItemTextClasses.secondary}`]: {
+        ...theme.typography.microText,
+        marginTop: "2px",
+        fontWeight: 500,
+        color: theme.palette.gray[50],
+      },
+
+      [`& .${listItemIconClasses.root}`]: {
+        color: theme.palette.gray[50],
+        minWidth: "unset",
+        marginRight: 12,
+      },
+
+      [`& .${listItemAvatarClasses.root}`]: {
+        border: "2px solid transparent",
+        marginRight: "12px",
+        borderRadius: "50%",
+        minWidth: "unset",
+      },
```

Comment:
This seems like styling specific to the workspace switcher, can we define it there instead of for all `MenuItem` components?